packages/banner/src/presenters/IconBackground/IconBackground.js

```diff
@@ -0,0 +1,38 @@
+import React from "react";
+import { ThemeContext } from "@hig/themes";
+import Icon, { names as iconNames, sizes as iconSizes } from "@hig/icon";
+
+import "./icon-background.scss";
+import classNames from "../classNames";
+import { types } from "../../types";
+
+/** @type {Object.<string, string>} */
+const iconNamesByType = {
+  [types.PRIMARY]: iconNames.INFO,
+  [types.COMPLETE]: iconNames.COMPLETE,
+  [types.WARNING]: iconNames.ISSUE,
+  [types.URGENT]: iconNames.ERROR
+};
+
+/**
+ * @typedef {Object} IconProps
+ * @property {string} type
+ */
+
+/**
+ * @param {IconProps} props
+ * @returns {JSX.Element}
+ */
+export function IconBackground({ type }) {
+  return (
+    <ThemeContext.Consumer>
+      {({ themeClass }) => (
+        <figure className={[classNames.iconBackground, themeClass].join(" ")}>
```

Comment:
Lets use the `classnames` package here. Then we won't have those extra spaces added on the end.

Reply:
I agree... tomorrow 👼